### ISSUE TITLE
[Feature Request]: Cluster autoscaler labels/taints per nodepool

### ISSUE THREAD

**Issue by Silvest89** [kube-hetzner/terraform-hcloud-kube-hetzner#982]
### Description

Would be nice to be able to set different taints/labels per autoscaler nodepool

**Comment by ricristian** [comment]
You already have this option in kube.tf.example or I am missing something ? 

` # Add extra labels on nodes started by the Cluster Autoscaler
  # This argument is not used if autoscaler_nodepools is not set, because the Cluster Autoscaler is installed only if autoscaler_nodepools is set
  # autoscaler_labels = [
  #   "node.kubernetes.io/role=peak-workloads"
  # ]

  # Add extra taints on nodes started by the Cluster Autoscaler
  # This argument is not used if autoscaler_nodepools is not set, because the Cluster Autoscaler is installed only if autoscaler_nodepools is set
  # autoscaler_taints = [
  #   "node.kubernetes.io/role=specific-workloads:NoExecute"
  # ]
`

**Comment by Silvest89** [comment]
@ricristian Yes but if you set it there it goes for all autoscaler node pools.

Something like this would be nice
```
  autoscaler_nodepools = [
    {
      name        = "autoscaled-small"
      server_type = "cpx11"
      location    = "fsn1"
      min_nodes   = 0
      max_nodes   = 1
      labels = []
      taints = []
    }
  ]
```

**Comment by Silvest89** [comment]
Implemented in https://github.com/kube-hetzner/terraform-hcloud-kube-hetzner/pull/1012
Waiting for upstream [autoscaler](https://github.com/kubernetes/autoscaler)
